src/app/frontend/replicasetlist/logsmenu.html

```diff
@@ -27,12 +27,15 @@
     </md-menu-item>
     <md-menu-item ng-repeat="pod in ctrl.replicaSetPodsList" class="kd-menu-logs-md-menu-item">
       <div flex class="kd-menu-logs-item">{{pod.name | middleEllipsis:10}}</div>
-      <div flex="50" class="kd-menu-logs-item">
-        <a>
+      <div flex="50" class="kd-menu-logs-item" ng-if="pod.podContainers[0].name != undefined">
```

Comment:
Hmm... Dont put logic in html files. Move this and following line to the controller. E.g., create a function for the ng-if and ng-href.